palace/fem/fespace.hpp

```diff
@@ -115,6 +123,48 @@ class FiniteElementSpace
                            mfem::Geometry::Type geom, const std::vector<int> &indices,
                            bool is_interp = false, bool is_interp_range = false);
 
+  template <typename VecType>
+  auto &GetTVector() const
```

Comment:
Seconded that this change is likely somewhat short-lived before adopting https://github.com/mfem/mfem/pull/4065, so I think leaving the naming as is makes the most sense. In case the `TVector` part of the name was being confused for temporary, this naming comes from https://libceed.org/en/latest/libCEEDapi/ (differentiated versus, fr example, an L-vector).